ansible/roles/artemis/tasks/main.yml

```diff
@@ -237,7 +237,14 @@
   replace:
     path: /opt/{{ artemis_name }}-broker/etc/artemis.profile
     regexp: 'Xmx2G'
-    replace: 'Xmx48G'
+    replace: 'Xmx16G'
```

Comment:
??

Reply:
This change is not needed.

Reply:
This is preferred to have some apple 2 spoke comparisons...G1 isn't that good with huge heaps as it has been sold 🤣